embedding_matrix_api/app/main-service.py

```diff
@@ -0,0 +1,91 @@
+# main-service.py for execute EMC (Embedding Matrix Generator) service.
+# The main idea of this service is to run and list the output bucket of the DRS service to see when N embedding has been done, if the N results are not there, the service does  nothing else. Once you have N embedding create an array (json object) with the results of the DRS service.
+
+
+import os
+import sys
+import time
+from minio import Minio
+import urllib3
+import io
+import ast
+import json
+
+#Function to create object with data in json format
+class format:
```

Comment:
@vicente87 Python class names should follow [CapWords convention](https://peps.python.org/pep-0008/#class-names).

Reply:
@jaime-cespedes-sisniega   I already modified this to a more appropriate nam